packages/api/smart.go

```diff
@@ -79,7 +79,6 @@ func validateSmartContract(r *http.Request, data *apiData, result *prepareResult
 					}
 					if !found {
 						log.WithFields(log.Fields{"type": consts.NotFound, "signature": ret[1]}).Error("unknown signature")
-						err = fmt.Errorf(`%s is unknown signature`, ret[1])
```

Comment:
We cannot delete this err. It must be returned. Maybe to use 
return contract, nil, fmt.Errorf(`%s is unknown signature`, ret[1])
The best way is to add E_UNKNOWNSIGN in error.go and return 
return contract, ret[1], fmt.Errorf(`E_UNKNOWNSIGN`)